### PR TITLE
Add pipelines to static webapp.

### PR DESCRIPTION
**Pull request by mokainos**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: .github/workflows/cft-api-mgmt.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: push shutter page to cft-api-mgmt static webapp
```
Comment (timja) [inline .github/workflows/cft-api-mgmt.yaml:1]: this doesn't need shuttering

File: .github/workflows/dtsse-ardoq-adapter.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: push shutter page to dtsse-ardoq-adapter static webapp
```
Comment (timja) [inline .github/workflows/dtsse-ardoq-adapter.yaml:1]: this doesn't need shuttering

File: .github/workflows/gateway-ccd.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+name: push shutter page to gateway-ccd static webapp
```
Comment (timja) [inline .github/workflows/gateway-ccd.yaml:1]: I don't think this need shuttering

File: .github/workflows/hmi-apim.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+name: push shutter page to hmi-apim static webapp
```
Comment (timja) [inline .github/workflows/hmi-apim.yaml:1]: this doesn't need shuttering

File: .github/workflows/jui-redirect.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
+name: push shutter page to jui-redirect static webapp
```
Comment (timja) [inline .github/workflows/jui-redirect.yaml:1]: redirects don't need shutterring

File: .github/workflows/portal.yaml [hunk 6]
```diff
@@ -0,0 +1,31 @@
+name: push shutter page to portal static webapp
```
Comment (timja) [inline .github/workflows/portal.yaml:1]: what is portal?

File: .github/workflows/sds-api-mgmt.yaml [hunk 7]
```diff
@@ -0,0 +1,31 @@
+name: push shutter page to sds-api-mgmt static webapp
```
Comment (timja) [inline .github/workflows/sds-api-mgmt.yaml:1]: doesn't need shuttering

File: .github/workflows/portal.yaml (hunk 6 above)
Comment (mokainos) [reply .github/workflows/portal.yaml:1]: portal.pre-recorded-evidence.justice.gov.uk"

**Review by timja — APPROVED** [review]

**Pull request merged**